moveit_ros/planning_interface/move_group_interface/src/move_group_interface.cpp

```diff
@@ -833,13 +876,24 @@ class MoveGroupInterface::MoveGroupInterfaceImpl
     goal.controller_names = controllers;
 
     auto goal_handle_future = execute_action_client_->async_send_goal(goal, send_goal_opts);
+    if (callback_executor_.spin_until_future_complete(goal_handle_future,
+                                                      server_timeout.to_chrono<std::chrono::duration<double>>()) !=
+        rclcpp::FutureReturnCode::SUCCESS)
+    {
+      RCLCPP_ERROR(logger_, "MoveGroupInterface::execute() get sent call failed :(");
```

Comment:
You should make this error message more descriptive. Try to explain what failed and why

Reply:
You'll probably need to update all error messages like this